editor/src/core/shared/element-template.ts

```diff
@@ -1994,32 +2039,16 @@ export function getElementsByUIDFromTopLevelElements(
   return result
 }
 
-export type ChildOrAttribute = JSXElementChild | JSXAttribute
+export type ChildOrAttribute = JSXElementChild
 
 export function childOrBlockIsChild(
   childOrBlock: ChildOrAttribute,
 ): childOrBlock is JSXElementChild {
-  switch (childOrBlock.type) {
-    case 'JSX_CONDITIONAL_EXPRESSION':
-    case 'JSX_ELEMENT':
-    case 'JSX_ARBITRARY_BLOCK':
-    case 'JSX_TEXT_BLOCK':
-    case 'JSX_FRAGMENT':
-      return true
-    case 'ATTRIBUTE_VALUE':
-    case 'ATTRIBUTE_NESTED_ARRAY':
-    case 'ATTRIBUTE_NESTED_OBJECT':
-    case 'ATTRIBUTE_FUNCTION_CALL':
-    case 'ATTRIBUTE_OTHER_JAVASCRIPT':
-      return false
-    default:
-      const _exhaustiveCheck: never = childOrBlock
-      throw new Error(`Unhandled type ${JSON.stringify(childOrBlock)}`)
-  }
+  return true
```

Comment:
😀 I guess this is a `TODO`

Reply:
It can be bulldozed, I just kept it in so that I didn't have to delete it from a bunch of places for now.